CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Versions
 ========
 
+2024-08-31
```

Comment:
Back to the future 🤣

Reply:
![image](https://github.com/user-attachments/assets/f5df633f-242d-4943-ac94-a1e902ec04a1)